ReversePolishNotationCalculator/MF1_ReversePolishNotationReader.cs

```diff
@@ -0,0 +1,117 @@
+namespace ReversePolishNotation;
+
+public class ReversePolishNotationReader
+{
+    public static double ReadPolishNotation(string str)
+    {
+        //if num -> save to stack
+        //if operand -> perform between two top nums
+
+        string[] instructions = str.Split();
+        Stack<double> numbers = new Stack<double>();
+
+        foreach (var op in instructions)
+        {
+            //is num or operand?
+            (InstructionType,ushort) instructionTypeTuple = GetInstructionType(op);
```

Comment:
Více čitelné by bylo použít `(InstructionType Type, ushort Length)`. To lze potom používat jako klíčová slova odkazující na itemy v Tuple. Například `instructionTypeTuple.Type` by vrátilo Item1. 

Když se tohle použije, tak je potom zřetelnější, co dané hodnoty v Tuple reprezentují.